assessments/0002-notary-project.md

```diff
@@ -0,0 +1,195 @@
+| ---                                       | --- | --- | --- | --- | --- |
+| Communication methods documented          |     |     | ✅  |     |     |
+| Beginner friendly issue backlog           |     | ✅  |     |     |     |
+| “New contributor” getting started content | ✅  |     |     |     |     |
+| Project governance documentation          | ✅  |     |     |     |     |
+
+Criteria 
+- 1 = (Is not present or requires significant work)
+- 3 = (is present, but needs work)
+- 5 = (is executed extremely well or no improvement required) |
+
+### Comments & recommendations
+
+- Most of the issues in this section are a function of not having a website yet, and the project still being in a building phase.
+  - As the website build process starts, put together a skeleton IA that can be filled in as the project grows.
+
+- There are documented community communication methods, but it could be collected and expanded during the site development process. Prometheus has a good example of a community page: https://prometheus.io/community/. 
+
+- Regarding beginner friendliness, consider processes/documents for:
+   - Triaging docs issues
+   - Clearly marking a way for new contributors to make code or documentation contributions (i.e. a “good first issue” label), and define what makes a good first issue
+   - Set up a process for maintaining/pruning stale issues
+   - New contributors/your first contribution
+   - Letting new users know where to get help
+
+- Complete the governance work started here: https://github.com/notaryproject/notaryproject/issues/55 https://github.com/notaryproject/notaryproject/pull/78, and add it to a governance page (or section) on the website.
+
+
+## Final Recommendations
+
+Is it possible plan documentation along side the V2 development? Will the documentation be able to get its structure from the specifications being developed?
+
+### Website
+
+- [x] Create separate documentation repository: https://github.com/notaryproject/notaryproject.dev
+- [ ] Setup a [Hugo website](https://gohugo.io) with a [Docsy theme](https://www.docsy.dev/)
+- [ ] Setup Netlify for hosting/CI
+- [ ] Develop a [maintainers list](https://github.com/notaryproject/notaryproject.dev/blob/main/.github/settings.yml) for the documentation and its repository (this has been started https://github.com/notaryproject/notaryproject/issues/77)
+- [ ] Develop a maintenance plan for the documentation and its repository 
+- [ ] Follow the [CNCF website guidelines checklist](https://github.com/cncf/techdocs/blob/main/howto/website-guidelines-checklist.md) for other required elements
+
+_Note_: [Notary Project branding artwork](https://github.com/cncf/artwork/tree/master/projects/notary) exists. Other branding elements, like colour schemes and whatnot, will need to be developed.
+
+### Information Architecture
+
+The content to be written could have the following structure (as a start - modifications will need to be made as development continues):
```

Comment:
```suggestion
We recommend reorganizing your content in the following way: 
```

As a general note: your writing is very tentative and shy. Speak with more authority, in active voice and present tense. 

Common phrases for you are:
- "The ___ to be ___..." (future tense, we prefer present tense in technical writing) 
- "..could have.... as a start" (You are making a recommendation as an expert. There is no "could" or "as a start". Things may change as you go through the project, but when giving people your expertise speak with confidence and finality.
- ("..will need") -> passive voice, we prefer active voice in technical writing. An easy way to test for passive voice is to add "... by dogs" at the end of the sentence. If it still makes sense, you're speaking in passive voice. For example:

"modifications will need to be made... by dogs" -> Bingo, passive voice!
versus
"As development continues this plan might change... by dogs" -> grammatically incorrect, we're speaking in active voice!

There's also no need to caveat that things might change as you go through development. This is an initial recommendation/plan - some modification as you progress is expected.